<?xml version="1.0" encoding="UTF-8"?>
<testsuite errors="1" failures="0" name="BubbleSort_tests.TestBubbleSort-20171101190349" skipped="0" tests="1" time="0.001">
	<testcase classname="BubbleSort_tests.TestBubbleSort" name="test_sample_test" time="0.001">
		<error message="global name 'assertEquals' is not defined" type="NameError">
<![CDATA[Traceback (most recent call last):
  File "/home/asha/eclipse-workspace/MyApp/myappdev/local/lib/python2.7/site-packages/mock/mock.py", line 1305, in patched
    return func(*args, **keywargs)
  File "/home/asha/eclipse-workspace/MyApp/src/unittest/python/BubbleSort_tests.py", line 13, in test_sample_test
    assertEquals(sort.inp, 5)
NameError: global name 'assertEquals' is not defined
]]>		</error>
	</testcase>
	<system-out>
<![CDATA[]]>	</system-out>
	<system-err>
<![CDATA[]]>	</system-err>
</testsuite>
